docs/source/developer-guide/localization.md

```diff
@@ -28,8 +28,13 @@ login.register=Registre
 ```
 Notice that the values to the right side of the equal sign have been translated but that the keys on the left side have not been changed. These keys must not change, because they will be referenced when oxAuth fetches the translated text.
 
-To add translation for not yet supported languages, just create new properties file in [resource](https://github.com/GluuFederation/oxAuth/tree/master/Server/src/main/resources) folder and name it messages_[language_code].properties, then add language code as supported-locale to the [faces-config.xml](https://github.com/GluuFederation/oxAuth/blob/master/Server/src/main/webapp/WEB-INF/faces-config.xml#L9).
-
 
+## How to apply localization in Gluu Server
 
+<!--
+To add translation for not yet supported languages, just create new properties file in [resource](https://github.com/GluuFederation/oxAuth/tree/master/Server/src/main/resources) folder and name it messages_[language_code].properties, then add language code as supported-locale to the [faces-config.xml](https://github.com/GluuFederation/oxAuth/blob/master/Server/src/main/webapp/WEB-INF/faces-config.xml#L9).
+-->
 
+- Get your properties file named "oxauth_LANGUAGE.properties" and put it inside `/opt/gluu/jetty/oxauth/custom/i18n`
```

Comment:
should the file name be `oxauth_<language_code>.properties`? And also give an example, like `oxauth_fr.properties` So  that there is no confusion.